### PR TITLE
Fixed survey register definition

### PR DESCRIPTION
**Pull request by SamGodwin2**
The old config was working when tested individually how it failed in concourse due to different tags being used, the PR ensues that the correct vars will be pulled in.

### CODE REVIEW

File: author.tf [hunk 1]
```diff
@@ -514,12 +514,12 @@ module "survey-register" {
   vpc_id                 = "${module.author-vpc.vpc_id}"
   aws_alb_arn            = "${module.author-eq-ecs.aws_external_alb_arn}"
   aws_alb_listener_arn   = "${module.author-eq-ecs.aws_external_alb_listener_arn}"
-  service_name           = "survey-register"
+  service_name           = "author-survey-register"
```
Comment (samiwel) [inline author.tf:517]: Is this change still required now that survey register has been removed from eq-terraform?

**Review by samiwel — APPROVED** [review]
LGTM, just a question about renaming the service.

File: author.tf (hunk 1 above)
Comment (SamGodwin2) [reply author.tf:517]: Not anymore, just thought I might as well make it consistent.
